README.md

```diff
@@ -1,100 +1,143 @@
 # Schematic
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FSage-Bionetworks%2Fschematic%2Fbadge%3Fref%3Ddevelop&style=flat)](https://actions-badge.atrox.dev/Sage-Bionetworks/schematic/goto?ref=develop) [![GitHub stars](https://img.shields.io/github/stars/Sage-Bionetworks/schematic)](https://github.com/Sage-Bionetworks/schematic/stargazers) [![GitHub forks](https://img.shields.io/github/forks/Sage-Bionetworks/schematic)](https://github.com/Sage-Bionetworks/schematic/network)
 
-## Usage
+## Introduction
 
-### Virtual Environment Setup
+SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. The Python based infrastructure provides a _novel_ schema-based, data ingress ecosystem, that is meant to streamline the process of metadata annotation and validation for various data contributors.
 
-Python 3 has built-in support for virtual environments (using `venv` module). Perform the following steps:
+## Installation Requirements and Pre-requisites
 
-After cloning the git repository, navigate into the `schematic` directory and run the command as below:
+Following are the tools or packages that you will need to setup `schematic` for your use:
 
-```bash
-python[3] -m venv .venv
-```
+- [`pyenv`](https://github.com/pyenv/pyenv)
+- Python 3.7.1 or higher
+- [`poetry`](https://github.com/python-poetry/poetry)
+
+Note: It is recommended to use `pyenv` to be able to easily switch between Python versions. Explicit switching between environments using `poetry` is possible as well.
 
-_Note: It is assumed that you are running all the below commands from the main/root (`schematic`) directory._
+Make sure you are a registered and certified user on [`synapse.org`](https://www.synapse.org/), and also have all the right permissions to download credentials files in the following steps.
 
-This creates a Python3 virtual environment (within the `root` folder/package), with its own site directories (isolated from the system site directories).
+## Package Setup
 
-To activate the virtual environment, run:
+### Clone Project Repository
```

Comment:
For example, I think it would be ideal if the user only had to do the following:
```
pip install schematic
schematic init  # This command would create `.synapseConfig` and `config.yml`
schematic manifest get --config config.yml --data_type <type> --dataset_id <synid>
```

Reply:
Right, eventually we will. But right now, since the package hasn't been registered with PyPI yet, our "users" would have to follow the same installation procedure(s) as our "contributors/developers", which is why I've put in the developer-focussed instructions in the main README (for now).

Reply:
Could we create section in the main README for the various types of users? 

- Contributors (us)
- DCC admins (_e.g._ managing Data Curator instance)
- Data curators (end users wanting to use CLI and installing from PyPI)

Reply:
The more I read this README, the more I think it would benefit from organizing by target audience. For example, the `python -m venv` command isn't relevant to contributors or DCC admins who use poetry, which does this for them automatically.